src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -1283,6 +1283,225 @@ describe('MDC-based MatSlider' , () => {
     }));
   });
 
+  describe('slider as a custom form control', () => {
+    let fixture: ComponentFixture<SliderWithFormControl>;
+    let testComponent: SliderWithFormControl;
+    let sliderInstance: MatSlider;
+    let inputInstance: MatSliderThumb;
+
+    beforeEach(waitForAsync(() => {
+      fixture = createComponent(SliderWithFormControl);
+      fixture.detectChanges();
+      testComponent = fixture.debugElement.componentInstance;
+      const sliderDebugElement = fixture.debugElement.query(By.directive(MatSlider));
+      sliderInstance = sliderDebugElement.componentInstance;
+      inputInstance = sliderInstance._getInput(Thumb.END);
+    }));
+
+    it('should not update the control when the value is updated', () => {
```

Comment:
Wouldn't the expectation be that the control is updated when the value changes?

Reply:
I'm not sure, but this test is a copy/pasted from the old slider.

https://github.com/angular/components/blob/master/src/material/slider/slider.spec.ts#L1396